portal/templates/patients_by_org.html

```diff
@@ -3,7 +3,11 @@
   <div id="patientList" class="reduce-font-sizes">
     <h4 class="tnth-headline">Patient List</h4>
     <br/>
-    <div><button id="createUserLink" onclick="location.replace('/patients/profile_create')" class="btn btn-lg">{{ _("Create a patient record") }}</button>&nbsp;&nbsp;<div class="or">or</div>&nbsp;&nbsp;<span class="profile-item-title">{{ _("Select a patient below to view or update details.") }}</span></div>
+    <div id="patientListOptions">
+      <button id="createUserLink" onclick="location.replace('/patients/profile_create')" class="btn btn-lg">{{ _("Create a patient record") }}</button>
```

Comment:
Would you be able to easily replace this with an HTML link like below? Using `url_for()` also allows us to change our APIs without having to change any template code :)

`<a href="{{ url_for('patients.profile_create') }}"></a>`